include/vdml/imubl.h

```diff
@@ -0,0 +1,210 @@
+/**
+ * \file pros/imubl.h
+ *
+ * IMU Backend Layer header for non-threadsafe functions for reset
+ * functions that interact with the registry.
```

Comment:
I'm curious what others think of this. I personally don't think it's worth it to bother exposing these. We don't really need to expose 3 ways of setting the offset... The getters I don't see a use for.

Reply:
in the process of reworking this to not have this at all.